specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2268,6 +2268,22 @@
             "type": "string",
             "x-ms-parameter-location": "method"
           },
+          {
```

Comment:
Don't complete this PR for now

Reply:
I've added the `DoNotMerge` label, please let me know when you're ready to proceed.

Reply:
OK, thank you!

Reply:
@jhendrixMSFT it's ready to proceed, thanks!